lib/fetch-requests/foods-fetch-requests.js

```diff
@@ -0,0 +1,25 @@
+const url = `https://protected-basin-11627.herokuapp.com/`
```

Comment:
Should we put a const like this somewhere else that would be available for the both of us to use?

Reply:
I think it's a bit early on to think about mass refactors, I'd like to watch the app play out a bit more before making decisions like this? I mean, I can see where it makes it easy to access early on, but to each their own, concerning refactor styles?

Reply:
I think this is good! I would have suggested looking into things like environment variables but those won't really play nice with github pages! So for now this is fine :)